.github/workflows/busted.yml

```
@@ -49,7 +49,7 @@ jobs:
 
       - name: Build ‘busted’ (bootstrap)
         run: |
-          luarocks make
+          luarocks make --pin
```

```suggestion
          luarocks make
```

Including a lock file is fine and many downstream situations should seriously consider using it. That being said it isn't right for every use case. This CI runner for one won't like this because we're testing a matrix of different Lua versions and dependency versions, so pinning will break all but one combination of the matrix.

---

that makes sense, but maybe you should have at least one flow to check the lockfile though